eventstore/views.py

```diff
@@ -20,6 +27,89 @@
     PrebirthRegistrationSerializer,
     PublicRegistrationSerializer,
 )
+from ndoh_hub.utils import validate_signature
+
+
+class MessagesViewSet(GenericViewSet):
+    def create(self, request):
+        validate_signature(request)
+        try:
+            webhook_type = request.META.get("HTTP_X_TURN_HOOK_SUBSCRIPTION", "whatsapp")
```

Comment:
Here, we're defaulting to a `whatsapp` webhook type if the header isn't included.

I don't think that this is the correct way to do this, we should rather send a 400 Bad Request error back if the header isn't specified. (and also add a unit test that tests this)